client/src/app.js

```diff
@@ -0,0 +1,144 @@
+import io from "socket.io-client";
+
+const ENDPOINT = "http://www.toonin.ml:8100/";
+
+const servers = {
+    iceServers: [
+      {
+        urls: [
+          "stun:stun.l.google.com:19302",
+          "stun:stun2.l.google.com:19302",
+          "stun:stun3.l.google.com:19302",
+          "stun:stun4.l.google.com:19302"
+        ]
+      }
+    ]
+};
+
+var socket;
+var audioElem;
+
+export function init(audioElement) {
+    audioElem = audioElement;
+    setSocketListeners = setSocketListeners.bind(this);
+    createAnswer = createAnswer.bind(this);
+}
+
+export function toonin() {
```

Comment:
Can we get rid of this function?

Reply:
No, because it sets the audioElem variable to hold reference to the audio tag in our HTML file. This is necessary to attach the received stream to the audio tag